src/Scaler.Demo/OrderGenerator/Dockerfile

```diff
@@ -1,16 +1,23 @@
-# https://hub.docker.com/_/microsoft-dotnet
+#See https://aka.ms/customizecontainer to learn how to customize your debug container and how Visual Studio uses this Dockerfile to build your images for faster debugging.
```

Comment:
Is this Dockerfile generated using Visual Studio?

Reply:
yes, the original was not working. So, had to overwrite with VS generated Docker.